product_video_link/__manifest__.py

```diff
@@ -20,6 +20,7 @@
     "depends": [
         "product",
         "base_video_link",
+        "sales_team",
```

Comment:
@bizzappdev Could you explain why you need this dependency ?

Reply:
In the security CSV file, the group from sales_team has been referenced, 
It seems like the group from sales_team is also referenced in v14, but dependency was not added.
The sales_team module is a base for the team management of the Sales and CRM modules, so either we have to add a dependency or change the group that will have access rights.